homework/src/index.js

```diff
@@ -31,17 +31,66 @@
   }
 
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
+    const root = document.getElementById("root");
+    const div = createAndAppend("div", root, { class: "container" });
+    const header = createAndAppend("p", div, {
+      text: "HYF Repositories",
+      class: "header"
+    });
+    const table = createAndAppend("table", root, { class: "table" });
+    const select = createAndAppend("select", header, { class: "select" });
+
+    const tbody = createAndAppend("tbody", table);
+    const tr = createAndAppend("tr", tbody);
+
+    createAndAppend("td", tr, tbody, { text: "Repository: ", class: "label" });
+    const repository = createAndAppend("a", tbody);
+    const description = createAndAppend("tr", tbody);
```

Comment:
Same here, you are creating a table row. It is not a 'description'. Make sure that the names you choose accurately reflect their content.